jython-processors/trigonometric-transformation/processor.json

```diff
@@ -45,15 +43,50 @@ The way to read the measure on the resulting trigonometric circle is then the sa
             "name": "periodical_column",
             "label": "periodiclal column",
             "type": "COLUMN",
-            "description": "The column containing a periodical information (ex: day of year, day of month, day of week, month....)",
+            "description": "Column containing a periodical information",
             "mandatory": true
         },
+        {
+            "name": "process_as_time_dimension",
+            "label":"Process as time dimension ?",
+            "type": "BOOLEAN",
+            "defaultValue" : true,
+            "description": ""
+        },
         {
             "name": "column_period",
             "label":"column period",
             "type": "DOUBLE",
             "defaultValue" : 365.24,
-            "description": "The period of the column containing the periodical information (ex: 365.24 for day of year, 30.43 for day of month, 7 for day of week,, 12 for month....)"
+            "description": "Periodical column period"
+        },
+        {
```

Comment:
reduce UX complexity: cos, sin, tan should all be output. The user can use the remove column processor if need be,